modules/ROOT/pages/getting-started.adoc

```diff
@@ -11,61 +11,73 @@ NOTE: For more information on configuration, refer to the documentation for xref
 
 To launch FCOS on AWS:
 
-. Go to the https://getfedora.org/coreos/download/[download page] to identify the correct AMI.
+. Identify the correct AMI image ID for your region from https://getfedora.org/coreos/download/[the download page]
 
-. Specify the Ignition configuration file as the https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html#instancedata-add-user-data[user-data]. You can skip this step, but you will want to specify a key using `--key-name` so that you can still access the node. This provides an easy way to test out FCOS without first creating an Ignition configuration.
+. Launch the VM using `aws ec2 run-instances`. The Ignition file can be passed to the VM as its https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html#instancedata-add-user-data[user data]. You can skip this step, but you will want to specify a key using `--key-name` so that you can still access the node. This provides an easy way to test out FCOS without first creating an Ignition configuration.
```

Comment:
Might be worth mentioning that the key used by `--key-name` has to be previously imported into AWS and is not a local file.

Reply:
Added!